### PR TITLE
Add Chinese Simplified translation for GPU saver

### PR DESCRIPTION
**Pull request by ninthseason**
as title

### CODE REVIEW

File: godsenttools_gpu_saver/resource/localization/zh-CN/godsenttools_shared.properties [hunk 1]
```diff
@@ -0,0 +1,2 @@
+godsenttools.name=GodSentTools
+godsenttools.enable=Enable
```
Comment (GitSparTV) [inline godsenttools_gpu_saver/resource/localization/zh-CN/godsenttools_shared.properties]: Why this is not translated?

File: godsenttools_gpu_saver/resource/localization/zh-CN/godsenttools_gpu_saver.properties [hunk 2]
```diff
@@ -0,0 +1,9 @@
+godsenttools.gpusaver.name=GPU Saver
```
Comment (GitSparTV) [inline godsenttools_gpu_saver/resource/localization/zh-CN/godsenttools_gpu_saver.properties]: You can translate this if this makes sense

File: godsenttools_gpu_saver/resource/localization/zh-CN/godsenttools_gpu_saver.properties [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+godsenttools.gpusaver.description="GPU Saver" 将在你的游戏窗口失去焦点或最小化时完全停止游戏渲染。 这有助于减少 GPU 负载以预防显存溢出.
+godsenttools.gpusaver.osxwarning=警告! 检测到 MacOS 系统, 该插件无法在此操作系统上自动工作.\n自动模式失效\n你可以通过 "手动启用" 按钮来停止渲染。
+godsenttools.gpusaver.enable.help=启动 GPU Saver, 若禁用，自动/手动模式将都无法工作。
+godsenttools.gpusaver.enable.manual=手动启用
+godsenttools.gpusaver.screen.enabled=GPU Saver 已启动.
+godsenttools.gpusaver.screen.presskey=按下任何键来激活渲染.
+godsenttools.gpusaver.screen.howtodisable=你可以通过以下路径禁用该功能:
+godsenttools.gpusaver.screen.wheretodisable=%s (Spawnmenu) > 设置 > GodSentTools > GPU Saver.
```
Comment (GitSparTV) [inline godsenttools_gpu_saver/resource/localization/zh-CN/godsenttools_gpu_saver.properties]: Spawnmenu can be translated

**Review by GitSparTV — CHANGES_REQUESTED** [review]
Few string are not translated

**Comment by ninthseason** [comment]
Some places have not been translated before because they are easy to cause ambiguity in the Chinese context. ( But now i have come up with another way to avoid it. )

For example, 'enabled the addon' and 'make the addon work right now' is difficult to distingush in Chinese.
We also say 'spawnmenu' seldomly, but 'Q menu' instead.

Now everything is going well. 😉

**Review by GitSparTV — APPROVED** [review]

**Pull request merged**